crates/markdown_preview/src/markdown_elements.rs

```diff
@@ -107,7 +122,7 @@ pub enum ParsedMarkdownTableAlignment {
 #[derive(Debug)]
 #[cfg_attr(test, derive(PartialEq))]
 pub struct ParsedMarkdownTableRow {
-    pub children: Vec<ParsedMarkdownText>,
+    pub children: Vec<Vec<MarkdownParagraph>>,
```

Comment:
Why have these code paths been changed to `Vec<Vec<_>>`?

Reply:
I have changed it just to make it render text and image together it's a bit hacky solution but on most cases length of inner vector will be 1